CONTRIBUTING.md

```diff
@@ -132,6 +132,19 @@ In order to create a new activity feature, create a new file in `lib/activity/`,
 The below diagram describes the lifecycle of an activity message delivery to a level of detail that is intended to give a good intuition of how activity features work.
 ![activity message delivery](https://user-images.githubusercontent.com/7718702/42683224-4d72e732-86bf-11e8-89eb-0311c1eace7b.png)
 
+### Unfurls
+
+Unfurls describe the set of features that are called "rich link previews" in user facing documentation. When you post a link to some github.com resources in Slack, the link will automatically "unfurl" showing you some information about the resource such as the title, body, state (open/closed), etc.
+
+Each unfurl consists of a few things:
+- A regular expression in the `routes` object in`lib/github-url` that matches the type of URL that you want to unfurl
```

Comment:
Maybe use the complete file name `lib/github-url.js` to be consistent. 

Maybe appending "that matches the type of URL that you want to unfurl to the resource" to help tie it into the next sentence.